spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -402,14 +404,16 @@ def not_ok?; false; end
               if order.current == :second
                 :second_access
               else
-                order.pass_to :second, :resume_on => :sleep
```

Comment:
Why are we messing with other specs here? Does this cause a regression elsewhere?